kubevirt/index.json

```diff
@@ -0,0 +1,32 @@
+{
+  "pathwayTitle": "KubeVirt",
+  "title": "KubeVirt Demo",
+  "description": "Deploy and use KubeVirt in a hosted sandboxed interactive environment",
```

Comment:
We can change the description to:
KubeVirt is a virtual machine management add-on for Kubernetes. Deploy it and play with it in a hosted sandboxed interactive environment.